app/scripts/templates/permissions.mustache

```diff
@@ -0,0 +1,30 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-confirm-header">{{#t}}%(serviceName)s would like to know...{{/t}}</h1>
```

Comment:
Do you want the id to be `fxa-permissions-header` or something different than `confirm`?